### PR TITLE
Improvement on basics community documentations

### PR DESCRIPTION
**Pull request by VaneskaSousa**
## Description:
Improvement on readme.md:
* Add a menu like in our website;
* Fix English grammar;
* Add KnowCode link and summary;
* Add list of our non-code contributors like OW2 asks for.

Others changes:
 * Add conduct code in OSS guidelines.

### CODE REVIEW

**Comment by VaneskaSousa** [comment]
Please, review, it take just a few minutes. I just wanna public it on telegram community

**Comment by ItaloYeltsin** [comment]
@nmarquesin review it =)

File: README.md [hunk 1]
```diff
@@ -19,12 +19,14 @@
 </div>
 
 ## What is TotalCross?
-
-TotalCross is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
+TotalCross is tools to make easy and fast GUI creation:
+ 
+* KnowCode:  A computer vision project to help developers to build their screen faster based on image made by design using a library to allow run android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;
```
Comment (risbakk) [inline README.md]: Please, change it to:
_* KnowCode:  A computer vision project to help developers build user interfaces in less time. A library that converts a design image to a file that runs android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;_

File: README.md [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
+* TotalCross SDK (or just TotalCross): Is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
```
Comment (risbakk) [inline README.md]: Please, change it to:
_* TotalCross SDK: Is a free open source GUI Creator and framework for developing cross-platform applications for Embedded Systems, Android, iOS, Wince, Windows and Linux desktop applications, and Linux Arm. Current supported languages are Kotlin, Java and any Java Based language._

File: README.md [hunk 3]
```diff
@@ -22,42 +22,42 @@
...
 
-* TotalCross Java API -  It is the combination of several apis such as GUI components, database and GPIO to create a single more robust API that guarantees quality and support in application development
-* TCVM. - The heart of totalcross sdk is present our virtual machine, originally idealized in a master's thesis, and already built and improved over 10 years. It's log-based (Java) architecture, bytecode "itself with its own folders" for the most frequent and implemented **almost 100% with C guarantees performance equivalent to native development**. To read more about the TCVM click here.
+* TotalCross Java API - Combination of several APIs such as GUI components, Database and GPIO to create a single more robust API that guarantees quality and support in application development.
+* TCVM -  Totalcross Virtual Machine, originally idealized as a Master's Thesis, has been built and improved over 10 years. It's log-based (Java) architecture, bytecode "itself with its own folders" for the most frequent and implemented **almost 100% with C guarantees performance equivalent to native development**. To read more about the TCVM click here.
```
Comment (risbakk) [inline README.md:38]: Please add link to "To read more about the TCVM click here" or delete that sentence.

File: README.md [hunk 4]
```diff
@@ -21,7 +21,7 @@
 ## What is TotalCross?
 TotalCross is tools to make easy and fast GUI creation:
  
-* KnowCode:  A computer vision project to help developers to build their screen faster based on image made by design using a library to allow run android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;
+* [KnowCode](https://github.com/TotalCross/KnowCodeXML):  A computer vision project that aims to help developers build their screens faster based on image made by designers and generate totocross screens ready to run on Android, iOS and Linux ARM devices
```
Comment (risbakk) [inline README.md:24]: Please, change it to:
* KnowCode: A computer vision project to help developers build user interfaces in less time. A library that converts a design image to a file that runs android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;

File: README.md [hunk 5]
```diff
@@ -1,4 +1,24 @@
...
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="http://learn.totalcross.com/" target="_blank">Docs</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://medium.com/totalcross-community" target="_blank">Blog</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://t.me/totalcrosscommunity" target="_blank">Telegram</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://www.youtube.com/c/totalcross" target="_blank">Videos</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://totalcross.com/community/" target="_blank">Community</a>
```
[the remark this answers is not in the document]
Comment (VaneskaSousa) [reply README.md:18]: this is right, the github preview dont show the text aligned but if you go in https://github.com/TotalCross/totalcross/blob/documentation/README.md you'll see it

File: README.md [hunk 6]
```diff
@@ -37,7 +37,8 @@ O TotalCross SDK is comprised by two main components:
 
 ## Community
 The TotalCross's big dream is to build a large and supportive community of enthusiastic mobile and embedded developers. You can be part of the [telegram](https://t.me/totalcrosscommunity) and here on [GitHub](https://github.com/totalcross/totalcross).
-### Developing for TotalCross
+
+### Be a TotalCross contributor
```
[the remark this answers is not in the document]
Comment (VaneskaSousa) [reply README.md:43]: done

File: README.md [hunk 7]
```diff
@@ -58,6 +59,13 @@ In order to create a better tool and develop GUI for embedded systems and develo
 
 ##### 👉 [See how to be a contributor](CONTRIBUTING.md)!
 
+### Your contributors
```
[the remark this answers is not in the document]
Comment (VaneskaSousa) [reply README.md:64]: done

File: README.md (hunk 2 above)
Comment (VaneskaSousa) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (VaneskaSousa) [reply README.md:38]: added

File: README.md (hunk 4 above)
Comment (VaneskaSousa) [reply README.md:24]: done

File: README.md [hunk 8]
```diff
@@ -19,12 +19,14 @@
 </div>
 
 ## What is TotalCross?
-
-TotalCross is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
+TotalCross is tools to make easy and fast GUI creation:
```
[the remark this answers is not in the document]
Comment (VaneskaSousa) [reply README.md]: done

**Pull request merged**